src/models/server/useServerStatus.js

```diff
@@ -105,7 +105,7 @@ export default function useServerStatus() {
     const fetchData = async () => {
       try {
         const result = await axios(
-          'https://kaiju.dyn.wildme.io:5005/api/engine/job/status/',
+          'https://kaiju.dyn.wildme.io:5010/api/engine/job/status/',
```

Comment:
I feel like this should eventually come from site settings or environment variables or somewhere not hard-coded like this.

Reply:
It will, DEX-1178. This is just a temporary, quick fix.